_config/site.yml

```diff
@@ -98,6 +98,7 @@ profiles:
     js_minifier: disabled
     html_minifier: disabled
     legacy_post_syntax_highlighting: disabled
+    ignore_older_than_days: 30
```

Comment:
Should be possible. Feel free to change the option if you like. Otherwise just make sure you blog enough so there are plenty of posts in the last 30 days ;)